app/models/promotional-discount.ts

```diff
@@ -32,6 +32,10 @@ export default class PromotionalDiscountModel extends Model {
     return this.type === 'signup';
   }
 
+  get isFromStageTwoCompletion() {
```

Comment:
```suggestion
  get isFromStage2Completion() {
```

(We do use numbers in variable names in other places, can stay consistent - examples are `canSeeSplitUpGitCommandsForStage1`, `canSeeLanguageGuideBeforeSolutionForStage2`)